### PR TITLE
Design modification as given by Vijayan

### PR DESCRIPTION
**Pull request by shashankag-microland**
Hi Bhanu,

As requested, I created the separate branch for analytics page and I request you to please look into it.

### CODE REVIEW

File: src/containers/Analytics/AnalyticsProbabilityComponent/AnalyticsProbabilityComponent.tsx [hunk 1]
```diff
@@ -1,24 +1,129 @@
...
+  shouldComponentUpdate(nextProps, nextState) {
+    return  (
+      this.props.analyticalProbabilityInfo !== nextProps.analyticalProbabilityInfo || this.props.analyticalCountInfo !== nextProps.analyticalCountInfo || this.state.value !== nextState.value
+    );
+  }
 
-class AnalyticsProbabilityComponent extends Component {
   render() {
+    const { classes } = this.props;
+    console.log(this.state.value);
```
Comment (bhanutejk-microland) [inline src/containers/Analytics/AnalyticsProbabilityComponent/AnalyticsProbabilityComponent.tsx:77]: remove the log

File: src/containers/Analytics/AnalyticsProbabilityComponent/AnalyticsProbabilityComponent.tsx [hunk 2]
```diff
@@ -1,24 +1,129 @@
...
+            <XyChartColumnSeries analyticalInfo={this.props.analyticalCountInfo} />
           </Grid>
         </Grid>
       </Fragment>
     );
   }
 }
 
-export default AnalyticsProbabilityComponent;
+export default withStyles(useStyles)(AnalyticsProbabilityComponent);
```
Comment (bhanutejk-microland) [inline src/containers/Analytics/AnalyticsProbabilityComponent/AnalyticsProbabilityComponent.tsx:129]: Don't use the react hooks

File: src/containers/Analytics/AnalyticsProbabilityComponent/AnalyticsProbabilityComponent.tsx (hunk 1 above)
Comment (shashankag-microland) [reply src/containers/Analytics/AnalyticsProbabilityComponent/AnalyticsProbabilityComponent.tsx]: I removed the console.log in new pushed code

File: src/containers/Analytics/AnalyticsProbabilityComponent/AnalyticsProbabilityComponent.tsx (hunk 2 above)
Comment (shashankag-microland) [reply src/containers/Analytics/AnalyticsProbabilityComponent/AnalyticsProbabilityComponent.tsx]: I removed the react hooks in new pushed files

**Pull request merged**